src/applications/vaos/containers/DateTimeSelectPage.jsx

```diff
@@ -129,7 +132,13 @@ export class DateTimeSelectPage extends React.Component {
           onChange={newData => {
             this.props.updateFormData(pageKey, uiSchema, newData);
           }}
-          formContext={{ availableSlots, availableDates, preferredDate }}
+          formContext={{
```

Comment:
We might consider in the future making the calendar widget a connected component, to avoid threading these props through formContext here. Not really sure if that's better or not.

Or going further, I wonder if we really get much out of this page using `SchemaForm` at all.

Reply:
Yeah, I agree this is a bit messy.  When I first started the calendar I thought that using it within the `SchemaForm` was necessary, but I guess we could have just updated the form data state with actions instead.